go-controller/pkg/factory/factory.go

```diff
@@ -1659,6 +1659,8 @@ type waitForCacheSyncer interface {
 }
 
 func waitForCacheSyncWithTimeout(factory waitForCacheSyncer, stopCh <-chan struct{}) map[reflect.Type]bool {
+	// Give some small for sync. It helps significantly reduce unit tests time
```

Comment:
can you fix the comment here please?

Perhaps you meant... `Give some small time for sync`